src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -99,6 +99,15 @@ private enum Mode
         }
 
         /// <summary>
+        /// Address of the linked resource.
+        /// </summary>
+        /// <remarks>
+        /// Passed through to the generated HTML in all cases.
+        /// </remarks>
+        [HtmlAttributeName(HrefAttributeName)]
+        public string Href { get; set; }
```

Comment:
must be bound because helper needs the un-encoded value and encoding semantics must be identical to `asp-fallback-href` and so on.  same for `ScriptTagHelper.Src`.